src/ui/components/Video/RecordedCursor.tsx

```diff
@@ -58,6 +67,11 @@ export function RecordedCursor() {
         display: "none",
         position: "absolute",
       }}
+      data-test-name="recorded-cursor"
+      data-cursordisplay="false"
+      data-clickdisplay="false"
+      data-clientx="0"
+      data-clienty="0"
```

Comment:
nit: Would be more consistent with how we name data attributes elsewhere in the codebase if you separated by "-", maybe

```suggestion
      data-client-x="0"
      data-client-y="0"
      data-show-click="false"
      data-show-cursor="false"
      data-test-name="recorded-cursor"
```